README.md

```diff
@@ -49,6 +56,10 @@ cd ios && pod install
         ```
         npx react-native-release-profiler --local <path to profile>
         ```
+    - On **web**:
+        ```
+        npx react-native-release-profiler --local <path to profile>
+        ```
```

Comment:
if this is the same on iOS, do we really want it to be a separate point, or do we want to say On iOS/Web?